src/Turbo/src/Twig/TurboRuntime.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\Twig;
+
+use Psr\Container\ContainerInterface;
+use Symfony\UX\Turbo\Bridge\Mercure\TopicSet;
+use Twig\Environment;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Pierre Ambroise <pierre27.ambroise@gmail.com>
+ *
+ * @internal
+ */
+class TurboRuntime implements RuntimeExtensionInterface
+{
+    public function __construct(
+        private ContainerInterface $turboStreamListenRenderers,
+        private string $default,
```

Comment:
```suggestion
        private readonly string $defaultTransport,
```

? (and update code below)